docs/src/pages/[platform]/components/link/LinkPropControls.tsx

```diff
@@ -20,19 +27,29 @@ export const LinkPropControls: LinkPropControlsInterface = ({
   setIsExternal,
   textDecoration,
   setTextDecoration,
+  children,
+  setChildren,
 }) => {
   return (
     <Flex direction="column">
+      <TextField
+        value={children as string}
+        onChange={(event) => setChildren(event.target.value)}
+        label="body"
+      />
       <TextField
         value={color as string}
         onChange={(e) => setColor(e.target.value)}
         label="color"
       />
-      <TextField
+      <SelectField
         value={textDecoration as string}
-        onChange={(e) => setTextDecoration(e.target.value)}
+        onChange={(event) => setTextDecoration(event.target.value)}
         label="textDecoration"
-      />
+      >
+        <option value="none">none</option>
```

Comment:
non-blocking (could be follow up): Should we include a few more options? https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration